php_phongo_classes.h

```diff
@@ -20,289 +20,317 @@
 #include "php_phongo_structs.h"
 
 /* Export zend_class_entry dependencies, which are initialized in MINIT */
-extern zend_class_entry *php_phongo_date_immutable_ce;
-extern zend_class_entry *php_phongo_json_serializable_ce;
+extern zend_class_entry* php_phongo_date_immutable_ce;
+extern zend_class_entry* php_phongo_json_serializable_ce;
 
 #if PHP_VERSION_ID >= 70000
 
-static inline php_phongo_bulkwrite_t* php_bulkwrite_fetch_object(zend_object *obj) {
-    return (php_phongo_bulkwrite_t *)((char *)obj - XtOffsetOf(php_phongo_bulkwrite_t, std));
+static inline php_phongo_bulkwrite_t* php_bulkwrite_fetch_object(zend_object* obj)
+{
+	return (php_phongo_bulkwrite_t*)((char*)obj - XtOffsetOf(php_phongo_bulkwrite_t, std));
 }
-static inline php_phongo_command_t* php_command_fetch_object(zend_object *obj) {
-    return (php_phongo_command_t *)((char *)obj - XtOffsetOf(php_phongo_command_t, std));
+static inline php_phongo_command_t* php_command_fetch_object(zend_object* obj)
```

Comment:
Unrelated to formatting changes, I wonder if these functions should have line breaks between them. Are they currently considered members of the same "paragraph"?